### PR TITLE
 update ltp tests

### PR DESCRIPTION
**Pull request by salsal97**

### CODE REVIEW

File: tests/ltp/README.md [hunk 1]
```diff
@@ -1052,7 +1052,7 @@ FAILING
 | /ltp/testcases/kernel/syscalls/sendmsg/sendmsg01 | 0 | 0 | 0 |
 | /ltp/testcases/kernel/syscalls/sendmsg/sendmsg02 | 0 | 0 | 0 |
 | /ltp/testcases/kernel/syscalls/sendmsg/sendmsg03 | 0 | 0 | 0 |
-| /ltp/testcases/kernel/syscalls/sendto/sendto01 | 1 | 1 | 1 |
+| /ltp/testcases/kernel/syscalls/sendto/sendto01 | 0 | 0 | 0 |
```
Comment (bodzhang) [inline tests/ltp/README.md:1055]: Did sendto01 test pass in the past?

File: tests/ltp/README.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ FAILING
 | /ltp/testcases/kernel/syscalls/alarm/alarm07 | 0 | 0 | 0 |
 | /ltp/testcases/kernel/syscalls/bind/bind01 | 0 | 0 | 0 |
 | /ltp/testcases/kernel/syscalls/bind/bind02 | 0 | 0 | 0 |
-| /ltp/testcases/kernel/syscalls/bind/bind03 | 1 | 0 | 0 |
+| /ltp/testcases/kernel/syscalls/bind/bind03 | 0 | 0 | 0 |
```
Comment (bodzhang) [inline tests/ltp/README.md:59]: Did bind03 test on ext2fs pass in the past?

File: tests/ltp/README.md [hunk 3]
```diff
@@ -132,7 +132,7 @@ FAILING
 | /ltp/testcases/kernel/syscalls/cma/process_vm_readv02 | 0 | 0 | 0 |
 | /ltp/testcases/kernel/syscalls/cma/process_vm_readv03 | 0 | 0 | 0 |
 | /ltp/testcases/kernel/syscalls/cma/process_vm_writev02 | 0 | 0 | 0 |
-| /ltp/testcases/kernel/syscalls/connect/connect01 | 1 | 1 | 1 |
+| /ltp/testcases/kernel/syscalls/connect/connect01 | 0 | 0 | 0 |
```
Comment (bodzhang) [inline tests/ltp/README.md:135]: Did connect01 test pass in the past?

File: tests/ltp/README.md [hunk 4]
```diff
@@ -439,7 +439,7 @@ FAILING
 | /ltp/testcases/kernel/syscalls/getitimer/getitimer02 | 0 | 0 | 0 |
 | /ltp/testcases/kernel/syscalls/getitimer/getitimer03 | 1 | 1 | 1 |
 | /ltp/testcases/kernel/syscalls/getpagesize/getpagesize01 | 1 | 1 | 1 |
-| /ltp/testcases/kernel/syscalls/getpeername/getpeername01 | 1 | 1 | 1 |
+| /ltp/testcases/kernel/syscalls/getpeername/getpeername01 | 0 | 0 | 0 |
```
Comment (bodzhang) [inline tests/ltp/README.md:442]: Did getpeername01 test pass in the past?

File: tests/ltp/README.md [hunk 5]
```diff
@@ -701,7 +701,7 @@ FAILING
 | /ltp/testcases/kernel/syscalls/mkdir/mkdir04 | 0 | 1 | 0 |
 | /ltp/testcases/kernel/syscalls/mkdir/mkdir05 | 1 | 1 | 1 |
 | /ltp/testcases/kernel/syscalls/mkdir/mkdir09 | 0 | 0 | 0 |
-| /ltp/testcases/kernel/syscalls/mkdirat/mkdirat01 | 1 | 1 | 1 |
+| /ltp/testcases/kernel/syscalls/mkdirat/mkdirat01 | 0 | 0 | 0 |
```
Comment (bodzhang) [inline tests/ltp/README.md:704]: Did mkdirat01 test pass in the past?

File: tests/ltp/README.md [hunk 6]
```diff
@@ -714,7 +714,7 @@ FAILING
 | /ltp/testcases/kernel/syscalls/mknod/mknod09 | 0 | 0 | 0 |
 | /ltp/testcases/kernel/syscalls/mknodat/mknodat01 | 0 | 0 | 0 |
 | /ltp/testcases/kernel/syscalls/mknodat/mknodat02 | 0 | 0 | 0 |
-| /ltp/testcases/kernel/syscalls/mlock/mlock01 | 1 | 1 | 1 |
+| /ltp/testcases/kernel/syscalls/mlock/mlock01 | 0 | 1 | 1 |
```
Comment (bodzhang) [inline tests/ltp/README.md:717]: Did mlock01 test on ext2fs pass in the past?

File: tests/ltp/ext2fs_fs_tests_passed.txt [hunk 7]
```diff
@@ -134,10 +134,10 @@
 /ltp/testcases/kernel/syscalls/symlinkat/symlinkat01
 /ltp/testcases/kernel/syscalls/truncate/truncate02
 /ltp/testcases/kernel/syscalls/truncate/truncate02_64
+/ltp/testcases/kernel/syscalls/unlinkat/unlinkat01
```
Comment (jxyang) [inline tests/ltp/ext2fs_fs_tests_passed.txt:137]: Can we add back the ltp tests disabled before the release? PR https://github.com/deislabs/mystikos/pull/621 is supposed to fix the regressions.

File: tests/ltp/README.md (hunk 2 above)
Comment (salsal97) [reply tests/ltp/README.md:59]: Yes it did, there was another PR to remove it

File: tests/ltp/README.md (hunk 3 above)
Comment (salsal97) [reply tests/ltp/README.md:135]: Same as bind03, there was anotherPR to remove it

File: tests/ltp/README.md (hunk 4 above)
Comment (salsal97) [reply tests/ltp/README.md:442]: Same as bind03, there was another PR to remove it

File: tests/ltp/README.md (hunk 5 above)
Comment (salsal97) [reply tests/ltp/README.md:704]: This passes now too, will fix this

File: tests/ltp/README.md (hunk 6 above)
Comment (salsal97) [reply tests/ltp/README.md:717]: Same as bind03, there was another PR to remove it

File: tests/ltp/README.md (hunk 1 above)
Comment (salsal97) [reply tests/ltp/README.md:1055]: Same as bind03, there was another PR to remove it

File: tests/ltp/ext2fs_fs_tests_passed.txt (hunk 7 above)
Comment (salsal97) [reply tests/ltp/ext2fs_fs_tests_passed.txt:137]: Yes. I think this PR has gotten a little out of date. Will run the script again and update

File: tests/ltp/README.md (hunk 2 above)
Comment (bodzhang) [reply tests/ltp/README.md]: @salsal97, is the root cause of the failure known?

**Review by jxyang — APPROVED** [review]

**Comment by jxyang** [comment]
@salsal97 Can you combine this with #641 to avoid potential merge conflicts?

File: tests/ltp/README.md (hunk 2 above)
Comment (salsal97) [reply tests/ltp/README.md]: Hi Bo, a couple of them were taken out at the same time, tracked here https://github.com/deislabs/mystikos/issues/599

Comment (salsal97) [reply tests/ltp/README.md]: And the PR details are here https://github.com/deislabs/mystikos/issues/562

**Pull request merged**